sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -2397,13 +2546,21 @@
           "id": "ewepno"
         },
         {
-          "html": "<p style=\"margin-left:0cm;\">Your application was subject to the federal and provincial financial need assessment calculations, based on Canada Student Loan Program and StudentAid BC’s respective policies</p><ul><li>Total family income on the previous year's returns: This Includes the income of your spouse, parents, stepparents, sponsors, or legal guardians</li><li>Scholarships and bursaries reported</li><li>Voluntary or targeted contributions reported, and</li><li>Assets reported by parents, stepparents, sponsors, or legal guardians (dependent students only)</li></ul>",
-          "label": "Content",
+          "label": "HTML",
+          "tag": "p",
+          "className": "",
+          "attrs": [
+            {
+              "attr": "",
+              "value": ""
+            }
+          ],
+          "content": "<p style=\\\"margin-left:0cm;\\\">Your application was subject to the federal and provincial financial need assessment calculations, based on Canada Student Loan Program and StudentAid BC’s respective policies</p><ul><li>Total family income on the previous year's returns: <b>${{ data.assessment?.totalFamilyIncome }}</b></li><li>This Includes the income of your spouse, parents, stepparents, sponsors, or legal guardians</li><li>Scholarships and bursaries reported</li><li>Voluntary or targeted contributions reported, and</li><li>Assets reported by parents, stepparents, sponsors, or legal guardians (dependent students only)</li></ul>",
```

Comment:
for margin, we can use bootstrap classes(https://getbootstrap.com/docs/4.0/utilities/spacing) instead of inline classes, if the class is not present you can create custom class

Reply:
Removed the style, as not needed